src/KingSystem/Quest/qstManager.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include <container/seadObjArray.h>
+#include <heap/seadDisposer.h>
+#include <prim/seadSafeString.h>
+#include "KingSystem/Quest/qstQuest.h"
+#include "KingSystem/Resource/resHandle.h"
+#include "agl/Utils/aglParameter.h"
+
+namespace ksys::qst {
+
+struct Quest;
+
+class Manager {
+    SEAD_SINGLETON_DISPOSER(Manager)
+public:
+    Manager();
+    virtual ~Manager();
+
+    void init(sead::Heap* heap);
+    void cleanUp();
+
+    void __auto0(act::Actor* actor);
```

Comment:
This should be renamed because any name that contains two underscores is reserved for the C++ implementation.